content/synthetics/uptime_check.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Uptime Check
+kind: documentation
+beta: true
+description: Simulate and monitor HTTP requests from specific locations
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+---
+
+<div class="alert alert-warning">Synthetics is currently in beta for the Datadog US Region. See the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics Request form</a>.</div>
+
+## Overview
+
+Uptime Checks are HTTP requests (GET, POST, PUT, etc.) executed by Datadog to your web properties or application endpoints at configurable periodic intervals from multiple locations around the world. These Checks verify not only that your applications are up and responding to requests, but that any conditions you define are met, such as response time, HTTP response code, and header or body contents.
```

Comment:
is properties the right word here?

Reply:
"These Checks" -> "These checks"

Reply:
"These Checks verify not only that your applications are up and responding to requests, but that any conditions you define are met, such as response time, HTTP response code, and header or body contents."

"These checks verify that your applications are responding to requests and verify that they meet any conditions you define, such as response time, HTTP response code, and header or body contents."